### PR TITLE
Added sex to key.

### PR DESCRIPTION
**Pull request by gerring**
Needed sex in the key to get the frames used in Tim's code.

### CODE REVIEW

**Review by tholland — DISMISSED** [review]

File: src/geneweaver/client/gedb.py [hunk 1]
```diff
@@ -203,31 +211,33 @@ def sort_for_concordance(
         """
         ret = {}
         for dr in expressions:
-            pvalue = dr.get(prop)
+            pvalue = dr.__dict__.get(prop)
```
Comment (bergsalex) [inline src/geneweaver/client/gedb.py]: This doesn't pass my smell test. I don't think you should need to access the internal __dict__.

File: tests/unit/__init__.py [hunk 2]
```diff
@@ -69,7 +70,10 @@ def search(self, drequest: DataRequest):
             # Read file and return json
             with gzip.open("tests/unit/imputations.json.gz", "rb") as f:
                 file_content = f.read()
-                return json.loads(file_content)
+                return [
+                    self._class_from_args(DataResult, item)
+                    for item in json.loads(file_content)
+                ]
```
Comment (bergsalex) [inline tests/unit/__init__.py]: Odd that we're doing this in the __init__.py files. Fixture definitions should really be in their own file, or in conftest.py.

File: src/geneweaver/client/gedb.py [hunk 3]
```diff
@@ -61,7 +67,9 @@ class DataResult:
     weights: List[float] = None  # noqa: N815
     geneIds: List[str] = None  # noqa: N815
     strains: List[str] = None  # noqa: N815
+    strain: str = None  # noqa: N815
```
Comment (bergsalex) [inline src/geneweaver/client/gedb.py:70]: What's the differtence between the singular and plural `strain`/`strains`?

File: src/geneweaver/client/gedb.py (hunk 1 above)
Comment (gerring) [reply src/geneweaver/client/gedb.py]: Okay, how do get a property value by name otherwise?

File: tests/unit/__init__.py (hunk 2 above)
Comment (gerring) [reply tests/unit/__init__.py]: Easy enough to move to conftest.py

File: src/geneweaver/client/gedb.py (hunk 3 above)
Comment (gerring) [reply src/geneweaver/client/gedb.py:70]: The type of search.

File: src/geneweaver/client/gedb.py (hunk 1 above)
Comment (gerring) [reply src/geneweaver/client/gedb.py]: getattr(...) just remembered

**Review by tholland — APPROVED** [review]

**Review by bergsalex — APPROVED** [review]

**Pull request merged**